07-MemoryManagement/user_space/myprog.c

```diff
@@ -0,0 +1,78 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <time.h>
+#include <sys/time.h>
+#include <math.h>
+
+int i, *p;
+struct timespec ts;
+long before, relalloc, relfree;
+long allocsize[5];
+
+int main(void)
+{
+	allocsize[0] = (long)pow(2, 10);
+	allocsize[1] = (long)pow(2, 15);
+	allocsize[2] = (long)pow(2, 22);
+	allocsize[3] = (long)pow(2, 32);
+	allocsize[4] = (long)pow(2, 35);
```

Comment:
No all required buffers are allocated.

Reply:
What do you mean?

Reply:
I understood, it was necessary from 0 to 64.